### ISSUE TITLE
Debian testing: nextcloud-client keeps running wizard on start-up

### ISSUE THREAD

**Issue by SigmundFreudsBartender** [nextcloud/client_theming#172]
I'm using nextcloud-client  on debian testing and it does not store my details, so that on each boot, it asks me for the server, user and psswd and starts a complete sync, too.

**Comment by enoch85** [comment]
Is this still an issue or was it just temporary?

**Comment by SigmundFreudsBartender** [comment]
Still happening, not on every boot but almost every time. Yes.

Best sfb

Am Sonntag 25. Juni 2017 schrieb Daniel Hansson:
> Is this still an issue or was it just temporary?
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/nextcloud/client_theming/issues/172#issuecomment-31092883

-- 
Gesendet von einem Jolla

**Comment by enoch85** [comment]
This sounds more like an ownCloud issue than a theming issue. 

@ivaradi any idea?

**Comment by SigmundFreudsBartender** [comment]
It didn't happen with the owncloud client i was using before. 

Am Montag 26. Juni 2017 schrieb Daniel Hansson:
> This sounds more like an ownCloud issue than a theming issue. 
> 
> @ivaradi any idea?
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/nextcloud/client_theming/issues/172#issuecomment-31093291

-- 
Gesendet von einem Jolla

**Comment by SigmundFreudsBartender** [comment]
Thing is: when i swap back to the oc-client everything works fine. May
not be the theming, but definitely the themed client.
best sfb

Am Sonntag, den 25.06.2017, 15:30 -0700 schrieb Daniel Hansson:
> This sounds more like an ownCloud issue than a theming issue.
> 
> @ivaradi any idea?
> 
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 
> 
> 
>   
>   
> 
> 
> 
> 
> 
> {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c5
> 5493e4bb","name":"GitHub"},"entity":{"external_key":"github/nextcloud
> /client_theming","title":"nextcloud/client_theming","subtitle":"GitHu
> b repository","main_image_url":"https://cloud.githubusercontent.com/a
> ssets/143418/17495839/a5054eac-5d88-11e6-95fc-
> 7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent
> .com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-
> b52498112777.png","action":{"name":"Open in
> GitHub","url":"https://github.com/nextcloud/client_theming"}},"update
> s":{"snippets":[{"icon":"PERSON","message":"@enoch85 in #172: This
> sounds more like an ownCloud issue than a theming issue.
> \r\n\r\n@ivaradi any idea?"}],"action":{"name":"View
> Issue","url":"https://github.com/nextcloud/client_theming/issues/172#
> issuecomment-310932917"}}}
-- 
xmpp: sfb@paranoici.org, sfb@joindiaspora.com

**Comment by ivaradi** [comment]
The configuration file should be ````~/.local/share/data/Nextcloud/nextcloud.cfg````. Could you disable auto starting the nextcloud client and check this file before and after the next boot if it exists and its contents look normal?

**Comment by SigmundFreudsBartender** [comment]
it just contains two lines:

[General]
optionalDesktopNotifications=true

[Accounts]
version=2

not normal?

Am Freitag, den 07.07.2017, 11:42 -0700 schrieb István Váradi:
> The configuration file should be
> ~/.local/share/data/Nextcloud/nextcloud.cfg. Could you disable auto
> starting the nextcloud client and check this file before and after
> the next boot if it exists and its contents look normal?
> 
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 
> 
> 
>   
>   
> 
> 
> 
> 
> 
> {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c5
> 5493e4bb","name":"GitHub"},"entity":{"external_key":"github/nextcloud
> /client_theming","title":"nextcloud/client_theming","subtitle":"GitHu
> b repository","main_image_url":"https://cloud.githubusercontent.com/a
> ssets/143418/17495839/a5054eac-5d88-11e6-95fc-
> 7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent
> .com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-
> b52498112777.png","action":{"name":"Open in
> GitHub","url":"https://github.com/nextcloud/client_theming"}},"update
> s":{"snippets":[{"icon":"PERSON","message":"@ivaradi in #172: The
> configuration file should be
> ````~/.local/share/data/Nextcloud/nextcloud.cfg````. Could you
> disable auto starting the nextcloud client and check this file before
> and after the next boot if it exists and its contents look normal?
> "}],"action":{"name":"View
> Issue","url":"https://github.com/nextcloud/client_theming/issues/172#
> issuecomment-313761879"}}}
-- 
xmpp: sfb@paranoici.org, sfb@joindiaspora.com

**Comment by ivaradi** [comment]
No, there should be more settings. Especially, the ````[Accounts]```` section should contain your account (login) data. Could you check if the directory and the file are writable by you? And their modification times?

**Comment by SigmundFreudsBartender** [comment]
Right, it looks better now. testing the updated nc-client know and come
to you when I know whether it works or not thank you so far,
sfb
Am Sonntag, den 09.07.2017, 07:45 +0000 schrieb István Váradi:
> No, there should be more settings. Especially, the [Accounts] section
> should contain your account (login) data. Could you check if the
> directory and the file are writable by you? And their modification
> times?
> 
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 
> 
> 
>   
>   
> 
> 
> 
> 
> 
> {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c5
> 5493e4bb","name":"GitHub"},"entity":{"external_key":"github/nextcloud
> /client_theming","title":"nextcloud/client_theming","subtitle":"GitHu
> b repository","main_image_url":"https://cloud.githubusercontent.com/a
> ssets/143418/17495839/a5054eac-5d88-11e6-95fc-
> 7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent
> .com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-
> b52498112777.png","action":{"name":"Open in
> GitHub","url":"https://github.com/nextcloud/client_theming"}},"update
> s":{"snippets":[{"icon":"PERSON","message":"@ivaradi in #172: No,
> there should be more settings. Especially, the ````[Accounts]````
> section should contain your account (login) data. Could you check if
> the directory and the file are writable by you? And their
> modification times?"}],"action":{"name":"View
> Issue","url":"https://github.com/nextcloud/client_theming/issues/172#
> issuecomment-313904853"}}}
-- 
xmpp: sfb@paranoici.org, sfb@joindiaspora.com

**Comment by SigmundFreudsBartender** [comment]
Now this is odd. Config-file looked good the other day and was working.
Today on boot it's completly empty.

**Comment by Djabal** [comment]
Same thing here, sometimes config file is erased...

**Comment by oxivanisher** [comment]
Same situation here. I am unable to trace the reason. :(
